src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        setColor(color);
```

Comment:
```suggestion
        super(color);
        this.radius = radius;
```
it's better to use constructor instead of setter here